Mage/src/main/java/mage/abilities/effects/common/ExileSpellWithTimeCountersEffect.java

```diff
@@ -18,16 +20,25 @@
 public class ExileSpellWithTimeCountersEffect extends OneShotEffect {
 
     private final int counters;
+    private final boolean gainsSuspend;
 
     public ExileSpellWithTimeCountersEffect(int counters) {
         super(Outcome.Exile);
         this.counters = counters;
+        this.gainsSuspend = false;
         this.staticText = "Exile {this} with " + CardUtil.numberToText(this.counters) + " time counters on it";
```

Comment:
Don't duplicate code. Generally only one constructor calls `super()`, and the other calls `this()` with default param, in this case it would be `this(counters, false)`